evap/staff/views.py

```diff
@@ -276,41 +276,51 @@ def warning_for_inapplicables(amount):
 
     @staticmethod
     def apply(
-        request, evaluations, email_template=None, email_template_contributor=None, email_template_participant=None
+        request,
+        evaluations,
+        email_template=None,
+        email_template_contributor=None,
+        email_template_participant=None,
+        delete_previous_answers=None,
     ):
         raise NotImplementedError
 
 
-class RevertToNewOperation(EvaluationOperation):
-    confirmation_message = gettext_lazy("Do you want to revert the following evaluations to preparation?")
+class ResetToNewOperation(EvaluationOperation):
+    confirmation_message = gettext_lazy("Do you want to reset the following evaluations to preparation?")
 
     @staticmethod
-    def applicable_to(evaluation):
-        return Evaluation.State.PREPARED <= evaluation.state <= Evaluation.State.APPROVED
+    def applicable_to(evaluation: Evaluation):
+        return evaluation.can_reset_to_new()
 
     @staticmethod
     def warning_for_inapplicables(amount):
         return ngettext(
-            "{} evaluation cannot be reverted, because it already started. It was removed from the selection.",
-            "{} evaluations cannot be reverted, because they already started. They were removed from the selection.",
+            "{} evaluation cannot be reset, because it is already published. It was removed from the selection",
+            "{} evaluations cannot be reset, because they were already published. They were removed from the selection.",
```

Comment:
```suggestion
            "{} evaluation cannot be reset, because it is already in preparation or published. It was removed from the selection",
            "{} evaluations cannot be reset, because they were already in preparation or published. They were removed from the selection.",
```